thirdparty/runtime_environments.go

```diff
@@ -244,10 +280,22 @@ func (c *RuntimeEnvironmentsClient) GetToolchains(ctx context.Context, opts Tool
 	defer resp.Body.Close()
 	if resp.StatusCode != http.StatusOK {
 		msg, _ := io.ReadAll(resp.Body)
-		return nil, errors.Errorf("HTTP request returned unexpected status '%s': %s", resp.Status, string(msg))
+		apiErr := errors.New(string(msg))
+		grip.Error(message.WrapError(apiErr, message.Fields{
+			"message":     "bad response code from image visibility API",
+			"reason":      "runtime environments API alerting",
+			"params":      params,
+			"status_code": resp.StatusCode,
+		}))
+		return nil, errors.Errorf("getting toolchains: %s", apiErr.Error())
 	}
 	toolchains := &APIToolchainResponse{}
 	if err := gimlet.GetJSON(resp.Body, &toolchains); err != nil {
+		grip.Error(message.WrapError(err, message.Fields{
+			"message": "parsing response from image visibility API",
+			"reason":  "runtime environments API alerting",
```

Comment:
actually tiny nit -- since this is consistent, I'd recommend making this a constant at the top of the file, like [this](https://github.com/evergreen-ci/evergreen/blob/42f82176ce76f1f688803d2c039b693f0cb252b6/thirdparty/github.go#L41)